smriprep/workflows/norm.py

```diff
@@ -239,7 +239,7 @@ def init_anat_norm_wf(
     norm_rpt.inputs.after_label = 'Participant'  # after
 
     ds_t1_2_tpl_report = pe.Node(
-        DerivativesDataSink(base_directory=reportlets_dir, keep_dtype=True),
+        DerivativesDataSink(base_directory=reportlets_dir, suffix='T1w'),
```

Comment:
@oesteban If you understand why we were getting effective inputs as listed [here](https://619-160736148-gh.circle-artifacts.com/0/tmp/ds054/work/smriprep_wf/single_subject_100185_wf/anat_preproc_wf/anat_norm_wf_MNI152Lin/ds_t1_2_tpl_report/_report/report.rst), let me know. I can't figure out where `suffix` is getting defined as `'t1w'`, so I'm just overriding it.

Reply:
weird. I'm okay overriding it. We can come back to this later.

Reply:
Unresolved. I think `DerivativesDataSink` must be overriding inputs, but it's really unclear where.

https://625-160736148-gh.circle-artifacts.com/0/tmp/ds054/work/smriprep_wf/single_subject_100185_wf/anat_preproc_wf/anat_norm_wf_MNI152Lin/ds_t1_2_tpl_report/_report/report.rst

Reply:
![weird](https://user-images.githubusercontent.com/598470/61074624-fcb6b900-a3cc-11e9-998c-068fca06bb17.png)

Reply:
Why this is not happening for ds005? - maybe there're some cached results?

Reply:
Are some of the working directories/derivatives directories cached?

Reply:
Yes - https://github.com/poldracklab/smriprep/blob/master/.circleci/config.yml#L428

We should re-run all the reportlet datasinks - i.e., remove the reportlets subfolder for both ds005 and ds054.